Neurobagel/assessment.json

```diff
@@ -0,0 +1,8334 @@
+[
+  {
+    "namespace_prefix": "snomed",
+    "namespace_url": "http://purl.bioontology.org/ontology/SNOMEDCT/",
+    "vocabulary_name": "SNOMED CT Assessment Scale",
+    "version": "1.0.0",
+    "terms": [
+      {
+        "id": "snomed:1303696008",
```

Comment:
I think you're missing some fields here like `abbreviation`, `description`, `same_as`, etc. - see the example terms file from https://github.com/neurobagel/annotation-tool/issues/161!

Also, I think we want to remove prefixes from the term IDs, similar to in the config.json.

Reply:
Ah, I see! That makes sense